src/lightning/fabric/strategies/fsdp.py

```diff
@@ -268,6 +270,30 @@ def broadcast(self, obj: TBroadcast, src: int = 0) -> TBroadcast:
         torch.distributed.broadcast_object_list(obj, src, group=_group.WORLD)
         return obj[0]
 
+    def clip_gradients_norm(  # type: ignore[override]
+        self,
+        module: "FullyShardedDataParallel",
+        optimizer: Optimizer,
+        max_norm: Union[float, int],
+        norm_type: Union[float, int] = 2.0,
+        error_if_nonfinite: bool = True,
+    ) -> None:
+        """Clip gradients by norm."""
+
+        rank_zero_warn("Gradient Clipping by Norm is currently experimental for FSDP. Proceed with Caution!")
```

Comment:
This warning is basically here, because we haven't yet figured out a proper way to test fsdp. So even though we make the calls as they should be, we cannot guarantee correctness here, which is why I added this warning.